backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/service/DefaultLocations.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * #%L
+ * de.metas.contracts
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.contracts.bpartner.service;
+
+import de.metas.bpartner.composite.BPartnerLocation;
+import lombok.Builder;
+import lombok.Value;
+
+import javax.annotation.Nullable;
+
+@Value
+@Builder
+public class DefaultLocations
+{
+	@Nullable
+	BPartnerLocation billToDefaultLocation;
+	boolean foundBillToDefaultLocation;
+
+	@Nullable
+	BPartnerLocation shipToDefaultLocation;
+	boolean foundShipToDefaultLocation;
```

Comment:
```suggestion
	pulbic boolean isFoundShipToDefaultLocation()
	{
		return shipToDefaultLocation != null;
	}
```

otherwise future devs might wonder under which circumstances it might make sense to have `foundShipToDefaultLocation == true` and `shipToDefaultLocation == null`

Reply:
done